packages/program/src/contract-call-script.ts

```diff
@@ -204,6 +216,129 @@ const getFunctionOutputInfos = (functionScopes: InvocationScopeLike[]): CallOutp
     };
   });
 
+/**
+ * Obtains script data for a contract call according to the V0 specification.
+ *
+ * @param call - the contract call to obtain for script data for.
+ * @param segmentOffset - the segment to generate pointers and offset data from.
+ * @returns the populated script data and call parameter offsets.
+ */
+export const getScriptDataV0 = (
```

Comment:
```suggestion
export const getScriptDataV0: ContractCallScriptFn = (
```

I think this is worth the type